compiler/rustc_codegen_gcc/src/intrinsic/mod.rs

```diff
@@ -66,6 +66,8 @@ fn get_simple_intrinsic<'gcc, 'tcx>(
         sym::log2f64 => "log2",
         sym::fmaf32 => "fmaf",
         sym::fmaf64 => "fma",
+        sym::fmuladdf32 => "fmaf", // NOTE: pessimal without FMA target feature
```

Comment:
Can you please explain what do you mean by "pessimal without FMA target feature"?

Reply:
Thanks, I'll update accordingly. [This comment](https://github.com/rust-lang/rust/pull/124874#discussion_r1593464410) on whether to fall back to fused or separate.